doc/nrf/known_issues.rst

```diff
@@ -244,6 +244,22 @@ Missing :file:`CMakeLists.txt`
 Thread
 ======
 
+.. rst-class:: v1-6-1 v1-6-0
+
+KRKNWK-10633: Link Metrics Enhanced ACK-based Probing does not work properly.
```

Comment:
```suggestion
KRKNWK-10633: When using Link Metrics Enhanced ACK-based Probing the Thread Information Element contains fixed data instead of the proper Link Metrics data assessed for the acknowledged frame.
```

Reply:
Fixed